automated/linux/ssuite/run-bench.sh

```diff
@@ -185,7 +186,13 @@ run_test() {
 	#
 	# <app_name>-startup--<workload_name>--<scheduler1_name> pass <real number> sec
 	#<app_name>-startup--<workload_name>--<scheduler1_name> fail
-	mv "${OUTPUT}/result_list.txt" "${RESULT_FILE}"
+
+	# mv "${OUTPUT}/result_list.txt" "${RESULT_FILE}"
```

Comment:
is this line necessary?

Reply:
@mwasilew Thanks for the review.

This line is not needed any more.
Deleting this line.